src/Base/PyTools.c

```diff
@@ -264,7 +270,11 @@ void PP_Fetch_Error_Text()
         PyObject* value = PyDict_GetItemString(errdata,"swhat");
         
         if (value!=NULL) {
-            strncpy(PP_last_error_info, PyString_AsString(value), MAX);
+#if PY_MAJOR_VERSION < 3
+            strncpy(PP_last_error_info, PyString_AsString(pystring), MAX);

```

Comment:
sry and thanks for fixing it so fast!
https://github.com/FreeCAD/FreeCAD/commit/e239a70ffda9e8f3ec8144c12ed631f9642036db